polls/signals.py

```diff
@@ -5,10 +5,12 @@
 from .models import AnnotatorProfile
 
 user = get_user_model()
+print("opened")
 
 
 @receiver(user_signed_up)
 def create_user_profile(request, user, *args, **kwargs):
+    print("hee")
```

Comment:
Do we really need these prints?

Reply:
My bad, it's for testing and I forgot to clean it out.